### PR TITLE
Style info_release page

### PR DESCRIPTION
**Pull request by mpanik**
screenshot-localhost_8082-2020 04 27-16_27_33

### CODE REVIEW

**Comment by mpanik** [comment]
@googlebot I signed it!

File: pages/ic/info_release.html [hunk 1]
```diff
@@ -26,76 +26,120 @@
...
+      <div id='information-release'
+           class='mdl-card mdl-shadow--2dp'>
+        <div class="mdl-card__title">
+          <h2 class="mdl-card__title-text">We need your permission</h2>
+        </div>
+        <div>
           <p>
+<<<<<<< HEAD
           You can goto <a href='{{.ConsentDashboardURL}}'>consent dashboard</a> to revoke your saved remember consent. All consents will expire in 90 days.
+=======
```
Comment (chaopeng) [inline pages/ic/info_release.html:47]: Please resolve the conflict. Thank you. This change is related to line 123 below

Comment (mpanik) [reply pages/ic/info_release.html]: Missed that one during rebasing. Should be fixed.

**Comment by chaopeng** [comment]
Seeing issues on low resolution.

1. on Mac 1080p (which on my external monitor), see screenshot 1, some texts are overlap
2. on more low resolution, linux 1536*864 (which we use on remote desktop), releasing item texts are overlap and the main div is bigger than page

I thing we need to fix issue 1, issue 2 maybe ok if it is hard to fix

**screenshot 1:**



**screenshot 2:**

**Comment by mpanik** [comment]
@chaopeng I did fix those overlapping labels, this should not be a problem anymore (let me know if is). I have also tried to improve responsiveness for smaller screens, but this is definitely not mobile friendly.

**Pull request closed without merging**